ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -160,7 +160,7 @@ object LedgerApiTestTool {
         new ColorizedPrintStreamReporter(
           System.out,
           config.verbose,
-        ).report(summaries)
+        ).report(summaries, identifierSuffix)
```

Comment:
I would recommend creating a type around this (e.g. `LedgerTestRunInfo`). We can add more information that needs to be fed to the reporter afterwards. In the meantime, it would be easier to understand what's the purpose of this.

Reply:
As long as we have just to parameters and type will be only use to pack and immediately unpack values inside, I'm in favour of sticking to 2 plain params. I agree that when we need to put more params here, or reuse somehere, case class should be used.